ember-client/src/test/scala/org/http4s/ember/client/internal/ClientHelpersSpec.scala

```diff
@@ -159,31 +157,31 @@ class ClientHelpersSpec extends Specification with CatsIO {
       } yield testResult
     }
 
-    "do not reuse when cancellation encountered running stream" in {
-      for {
-        reuse <- Ref[IO].of(Reusable.DontReuse: Reusable)
-
-        testResult <-
-          ClientHelpers
-            .postProcessResponse(
-              Request[IO](),
-              Response[IO](body = fs2
-                .Stream(1, 2, 3, 4, 5)
-                .map(_.toByte)
-                .zipLeft(
-                  fs2.Stream.awakeDelay[IO](1.second)
-                )
-                .interruptAfter(2.seconds)),
-              reuse
-            )
-            .use { resp =>
-              resp.body.compile.drain.attempt >>
-                reuse.get.map { case r =>
-                  r must beEqualTo(Reusable.DontReuse)
-                }
-            }
-      } yield testResult
-    }.pendingUntilFixed
+//    "do not reuse when cancellation encountered running stream" in {
+//      for {
+//        reuse <- Ref[IO].of(Reusable.DontReuse: Reusable)
+//
+//        testResult <-
+//          ClientHelpers
+//            .postProcessResponse(
+//              Request[IO](),
+//              Response[IO](body = fs2
+//                .Stream(1, 2, 3, 4, 5)
+//                .map(_.toByte)
+//                .zipLeft(
+//                  fs2.Stream.awakeDelay[IO](1.second)
+//                )
+//                .interruptAfter(2.seconds)),
+//              reuse
+//            )
+//            .use { resp =>
+//              resp.body.compile.drain.attempt >>
+//                reuse.get.map { case r =>
+//                  r must beEqualTo(Reusable.DontReuse)
+//                }
+//            }
+//      } yield testResult
+//    }.pendingUntilFixed
```

Comment:
any test expert knows how to fix that?

Reply:
Do we have an issue on this so it doesn't get forgotten, at least?

Reply:
here you go https://github.com/http4s/http4s/issues/4257